### ISSUE TITLE
Audio Tuner: Buttons should have better labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#242]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/webaudiotuner/

There are two sets of buttons with - or + as the visual label. This should probably be better for screen reader users. It may not read our great and it is impossible to tell the two sets apart when navigating via form controls.

An option is as follows:

* Add aria-label="increase" to the + and aria-label="decrease" to the - button. 
* Add IDs to the legend elements above, and the plus/minus buttons
* Add aria-labelledby="IDofPlusOrMinusButton IDofLegend" (with the correct ID's)

This would read out "increase base frequency".

Double check in accessibility tree to make sure the Name value is being set correctly.

**Comment by alrra** [comment]
Closing this in favor of https://github.com/MicrosoftEdge/Demos/pull/281.
